custom_components/powercalc/data/apple/MQJ83/model.json

```
@@ -0,0 +1,30 @@
+{
+    "calculation_enabled_condition": "{{ is_state('[[entity]]', 'playing') }}",
+    "calculation_strategy": "linear",
+    "device_type": "smart_speaker",
+    "linear_config": {
+        "calibrate": [
+            "10 -> 7.06",
+            "20 -> 7.14",
+            "30 -> 8.27",
+            "40 -> 11.74",
+            "50 -> 13.35",
+            "60 -> 15.29",
+            "70 -> 16.34",
+            "80 -> 18.84",
+            "90 -> 20.09",
+            "100 -> 27.08",
+            "0 -> 1.01"
+        ]
+    },
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plus Plug US",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.14:docker"
+    },
+    "name": "Apple HomePod Gen 2",
+    "standby_power": 7.01
```

```suggestion
    "standby_power": 1.01
```

Standby power was still too high.
Think we need this on 1.01 right?
I have made this as suggestion, you can just click apply to do the commit.

---

Yes, sorry, it should be 1.01.   I edited it manually and forgot to do that one.